requirements.txt

```diff
@@ -1,4 +1,5 @@
-flufl.enum 
+click
+flufl.enum
```

Comment:
can be removed if python 2.7 is not supported

Reply:
done.